### DISCUSSION TITLE
How do you get it to work on Macos?

### DISCUSSION THREAD

**Discussion by dazzng** [Purfview/whisper-standalone-win#90] [Q&A] [answered]
sorry, how do you use this with Macos? I am new to this.

**Comment by Purfview** [comment, marked as answer]
Download the executable for macOS from [Releases](https://github.com/Purfview/whisper-standalone-win/releases), extract it and use it in the terminal.

Here is a random guide I found on Youtube -> https://www.youtube.com/watch?v=FfT8OfMpARM 

Btw, Apple has officially dropped support for Nvidia GPUs in their newer operating systems, starting with macOS Mojave (10.14), so, it's not possible to run it on CUDA [GPU] in Mac OS. [ more reading about it there -> [forums.developer.nvidia.com](https://forums.developer.nvidia.com/t/when-will-the-nvidia-drivers-be-released-for-macos-10-15-catalina) ]

**Reply by dazzng** [reply]
thank you but where do I find the executable? Do I need to download it from here: https://github.com/guillaumekln/faster-whisper?

**Reply by Purfview** [reply]
Post was updated with the link:
Download the executable for macOS from [Releases](https://github.com/Purfview/whisper-standalone-win/releases)

**Reply by dazzng** [reply]
thank you:) I will probably have a few questions later on as well after I try to install it.

**Reply by Purfview** [reply]
It's not "installable", you just copy the executable where you want and that's it.

**Reply by dazzng** [reply]
yes sure that's what I meant. But let me ask a question. I downladed the macos file from Releases which was a zip file. I extracted it to my Downloads folder as a folder. Then I ran terminal inside it and try to run the command for whisper-faster.exe but it does not execute it.

**Reply by Purfview** [reply]
"exe" is for Windows, there is no extension on MacOS. If you run from the current folder and video/audio file is in same folder then use:
`./whisper-faster test2.mkv --language=English --model=medium`

**Reply by dazzng** [reply]
so this is what I am doing, correct me pls if I am doing wrong. I downloaded the zip file for Macos (Whisper-Faster_r160.1_macOS-x86-64). I extracted it into a folder then I have one folder in that and one license.txt. Inside the folder, I have only an exe for whisper-faster (which is for windows). Inside the main folder where license.txt is found, I run terminal and put this code: ./whisper-faster test2.mkv --language=English --model=medium

Is this it?

**Reply by Purfview** [reply]
>  I have only an exe for whisper-faster (which is for windows).

There is no Windows "exe" files in "Whisper-Faster_r160.1_macOS-x86-64"...

>  Inside the main folder where license.txt is found, I run terminal and put this code...

There is "Whisper-Faster" folder and in that folder there is executable "whisper-faster", so you need to go into that folder before you try to run that command.

**Reply by dazzng** [reply]
ok thanks I did that. then it started transcribing a 2 hour file. But it seems rather slow. It has been around 15 mns and it is only at 20th minute of the video. Am I doing sth wrong?

It also says Standalone Faster-Whisper r160.1 running on: CPU
and also this: Intel MKL WARNING: Support of Intel(R) Streaming SIMD Extensions 4.2 (Intel(R) SSE4.2) enabled only processors has been deprecated. Intel oneAPI Math Kernel Library 2025.0 will require Intel(R) Advanced Vector Extensions (Intel(R) AVX) instructions.

**Reply by Purfview** [reply]
> But it seems rather slow. It has been around 15 mns and it is only at 20th minute of the video. Am I doing sth wrong?

Looks normal speed for medium/large model on CPU. You can try `--beam_size=1` to make it faster, check if `--compute_type=float32` is faster for you.

> Intel MKL WARNING

Where it comes from? Anyway, it's just a warning, ignore it.

**Reply by dazzng** [reply]
thank you. Intel MKL Warning came from the terminal. just after the CPU thing.

**Reply by dazzng** [reply]
if I want to do this faster, would I have to run it on GPU? Which requires installing the Nvidia components?

**Reply by Purfview** [reply]
What is your GPU and size of VRAM?

**Reply by dazzng** [reply]
I am on Macbook pro m2 14 inch version

**Reply by Purfview** [reply]
> Macbook pro m2 14 inch version

Isn't this like from 2021-2023? The last Macs featuring an Nvidia GPU was in 2015.
CUDA works only with Nvidia GPU.

**Reply by dazzng** [reply]
yes it is. It is one of the most recent versions of macbook. So I can only work it out with CPU?

**Reply by Purfview** [reply]
Yes.

**Reply by dazzng** [reply]
is there a difference between the accuracy of the text produced by the CPU or GPU? or is it only about the speed of transciption?

**Reply by Purfview** [reply]
It's only about the speed.

**Reply by dazzng** [reply]
thank you for answering the questions and helping all the way along. If I ned something else, I will write here again.

**Reply by dazzng** [reply]
is it possible to eliminate the outside voices apart from the one who is speaking?

**Reply by Purfview** [reply]
> is it possible to eliminate the outside voices apart from the one who is speaking?

Only by editing the subtitles.

**Reply by dazzng** [reply]
ok thanks for the very quick response. One more thing, can you separate the text by the speaker if there are more than one?

**Reply by Purfview** [reply]
No. Read the help, there is written everything what you can do: `--help`.

**Reply by dazzng** [reply]
thanks

**Reply by azizrahip** [reply]
> Post was updated with the link: Download the executable for macOS from [Releases](https://github.com/Purfview/whisper-standalone-win/releases)

I cannot find any executable for macOS currently.

**Reply by Purfview** [reply]
> I cannot find any executable for macOS currently.

It hides under "Assets" button.
